docs/platform/howto/byoc/aws-privatelink-byoc.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Use AWS PrivateLink with BYOC services
+sidebar_label: Use AWS PrivateLink
+limited: true
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import RelatedPages from "@site/src/components/RelatedPages";
+
+Enable and manage AWS PrivateLink for your Aiven services deployed in your own cloud using [bring your own cloud (BYOC)](/docs/platform/concepts/byoc).
+
+## Limitations
+
+- AWS PrivateLink for Aiven BYOC is a feature with [limited availability](/docs/platform/concepts/service-and-feature-releases#limited-availability-).
+- You can enable AWS PrivateLink for your Aiven BYOC service only if it uses the [AWS BYOC
+  private deployment](/docs/platform/concepts/byoc#byoc-architecture) model.
+
+## Prerequisites
+
+- [Aiven CLI](/docs/tools/cli)
+- [AWS CLI](https://aws.amazon.com/cli/)
+- [Terraform](https://developer.hashicorp.com/terraform)
+- Optionally, access to:
+  - [Aiven Console](https://console.aiven.io/)
+  - [AWS Management Console](https://console.aws.amazon.com)
+
+## Set up permissions
+
+1. [Download the latest version of your Terraform template](/docs/platform/howto/byoc/download-infrastructure-template).
+
+   This template contains the necessary configuration to grant
```

Comment:
This is sort of repeated information in the sentence below the next step, so I think one of them could be removed. If it were me, I'd probably move this to the start of this section as an introduction explaining that they will use the TF template to given Aiven the necessary permissions.